File: PolicyManagement.Api/PolicyManagement.Models/Motor/PreviousPolicyFormDataModel.cs
using PolicyManagement.Dtos.Common;
using System;

namespace PolicyManagement.Models.Motor
{
    public class PreviousPolicyFormDataModel
    {
        public short LastYearInsuranceCompany { get; set; }
        public string PreviousPolicyNumber { get; set; }
        public string LastPolicyExpiryDateString { get; set; }
        public DateTime? LastPolicyExpiryDate { get; set; }
        public DateDto LastPolicyExpiryDateDto { get; set; }
    }
}

File: PolicyManagementAngular/src/app/app-entites/models/motor/previous-policy-form-data-model.ts
import { IDateDto } from "src/app/app-entites/dtos/common/date-dto";

export interface IPreviousPolicyFormDataModel {
    LastYearInsuranceCompany: number;
    PreviousPolicyNumber: string;
    LastPolicyExpiryDateString: string;
    LastPolicyExpiryDateDto: IDateDto|null;
    PreviousPolicyPlan?:string|null,
    PreviousPolicySumInsured?: number|null,
}